src/js/msp/MSPHelper.js

```diff
@@ -1793,8 +1815,10 @@ MspHelper.prototype.process_data = function(dataHandler) {
 
 /**
  * Encode the request body for the MSP request with the given code and return it as an array of bytes.
+ * The second (optional) 'modifierCode' argument can be used to extend/specify the behavior of certain MSP codes
+ * (e.g. 'MSPCodes.MSP2_GET_TEXT' and 'MSPCodes.MSP2_SET_TEXT')
  */
-MspHelper.prototype.crunch = function(code) {
+MspHelper.prototype.crunch = function(code, modifierCode = undefined) {
```

Comment:
this `undefined` is redundant, the only way `=` works in function arguments is if you don't pass anything or pass `undefined`. This is equivalent to:
```suggestion
MspHelper.prototype.crunch = function(code, modifierCode) {
```